src/UnitOfWork.php

```diff
@@ -1565,7 +1566,23 @@ final public static function getIdHashByIdentifier(array $identifier): string
 
         return implode(
             ' ',
-            $identifier,
+            array_map(
+                static function ($value) {
+                    if ($value instanceof BackedEnum) {
+                        return $value->value;
+                    }
```

Comment:
This looks redundant… isn't it already handled on line 1561?

Reply:
The code looks redundant yes, but the check is not ?

Reply:
I don't get it… after line 1565, can no longer contain any `BackedEnum`, right? So why add this check?

Reply:
Ok yes got it. I removed this check thank you !